rotkehlchen/db/updates.py

```diff
@@ -337,28 +335,40 @@ def update_global_addressbook(self, data: list[dict[str, Any]], version: int) ->
     def update_location_asset_mappings(self, data: dict[str, list[dict[str, Any]]], version: int) -> None:  # noqa: E501
         """Applies location asset mappings updates in the global DB"""
         log.info(f'Applying update for location asset mappings to v{version}')
-        for update_function, entry_type, raw_data_key in (
-            (GlobalDBHandler.add_location_asset_mappings, LocationAssetMappingUpdateEntry, 'additions'),  # noqa: E501
-            (GlobalDBHandler.update_location_asset_mappings, LocationAssetMappingUpdateEntry, 'updates'),  # noqa: E501
-            (GlobalDBHandler.delete_location_asset_mappings, LocationAssetMappingDeleteEntry, 'deletions'),  # noqa: E501
+        for update_query, entry_keys, raw_data_key in (
+            (
+                'INSERT OR IGNORE INTO location_asset_mappings(local_id, location, exchange_symbol) VALUES(?, ?, ?);',  # noqa: E501
+                ('asset', 'location', 'location_symbol'),
+                'additions',
```

Comment:
so the change is ignoring/replacing right? Does it make sense to change the logic in the `GlobalDBHandler`? Otherwise we will change the logic and forget about changing this. We could have a ignore_conflicts argument but don't know if it is tricking too much the logic. TBH I'm not sure what is better here